service-manual/making-software/security-testing.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: detailed-guidance
+title: Security testing
+subtitle: What is security testing and how to use it
+category: making-software
+type: guide
+audience:
+  primary: tech-archs, web-ops, qa
+  secondary: developers, performance-analysts
+phases:
+  - alpha
+  - beta
+  - live
+breadcrumbs:
+  -
+    title: Home
+    url: /service-manual
+  -
+    title: Making software
+    url: /service-manual/making-software
+---
+
+Security Testing intends to find flaws and vulnerabilities in an IT system, as well as the people and processes that surround the system. The practice includes:
+
+* formal security testing for live service accreditation, eg [penetration testing](https://www.gov.uk/service-manual/operations/penetration-testing.html), which aggressively attacks a system to check its defences, and [DDOS](http://en.wikipedia.org/wiki/Denial-of-service_attack) (Distributed Denial of Service) durability testing, which tries to make a service unavailable 
+* system feature testing, eg quality assurance testing of password functions
```

Comment:
I've left notes further down about the distinction here, but it doesn't feel quite right. We should also be clearly leading with solid security practice and then showing how that flows into assurance and accreditation processes rather than leading with the compliance aspects of security.